### PR TITLE
add a menu item to run audio stats script

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by ZappoMan** [comment]
Test plan:
- Developer > Audio > Stats... now runs the audio stats script to make it easier to find

**Comment by hifi-gustavo** [comment]
Build console output for commit: 76344f325334e4469e6c1fe7f2dbb57faca4fd86 available at: https://gist.github.com/d546149ea750f908f88d558fca58f9b1<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8929-76344f3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8929-76344f3.dmg

**Comment by kevinmthomasse** [comment]
testing

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -620,6 +620,14 @@ Menu::Menu() {
...
     MenuWrapper* audioDebugMenu = developerMenu->addMenu("Audio");
 
+    action = addActionToQMenuAndActionHash(audioDebugMenu, "Stats...");
+    connect(action, &QAction::triggered, [] {
+        auto scriptEngines = DependencyManager::get<ScriptEngines>();
+        QUrl defaultScriptsLoc = defaultScriptsLocation();
+        defaultScriptsLoc.setPath(defaultScriptsLoc.path() + "developer/utilities/audio/stats.js");
+        scriptEngines->loadScript(defaultScriptsLoc.toString());
+    });
+
```
Comment (AndrewMeadows) [inline interface/src/Menu.cpp]: Interesting idea... a hard coded menu option that launches a JS script from a known location.  The JS script in question loads a QML file that reads data from the AudioStats QOBJECT exposed to JS under that name (an AudioClient instance).  I'm not objecting, but I was wondering "Is this an instance of an existing pattern or something new?"  As far as I can tell this is new.

I seem to recall that Samcake had written some graphics engine debugging scripts that would render stats in overlays.  While they sounded interesting and that pattern could probably be used to render physics simulation stats, I have not yet risen to the challenge of manually hunting for those scripts and trying them out.  If they could be launched from a debug menu item then I'd be more likely to do it.

**Comment by kevinmthomasse** [comment]
## QA Test
**QA Tester**: KevinMThomas
**QA Pass**: **QA Pass** SUCCESS
**Additional Test Results + Observations**:  Success.  Audio stats menu works as well as the graphs.  This is perfect timing as I am set to perform in ```Good-Times``` tonight as this is critical to monitor.
capture

File: interface/src/Menu.cpp (hunk 1 above)
Comment (zzmp) [inline interface/src/Menu.cpp]: I think adding common debugging scripts to the developer menu is a great idea. It's a dev menu, after all - noone should be checking it for polish.

**Pull request merged**